geth/geth-genesis.json

```diff
@@ -13,7 +13,7 @@
     "muirGlacierBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 0,
-    "shardingForkBlock": 5,
+    "shardingForkBlock": 8,
```

Comment:
Lodestar has an upgrade schedule which includes Capella. We need them to all be the same.

Reply:
Doing this means the beacon chain upgrades EIP4844 before execution which breaks things.
Can we change lodestar's upgrade block to `5`? If not, you'll have to update the `EIP4844_FORK_EPOCH` such that the beacon chain upgrades at the same time.

Reply:
I changed EIP4844_FORK_EPOCH in the chain-config.

Lodestar has to upgrade through Capella.


https://github.com/Inphi/eip4844-interop/pull/52/files#diff-42abb036bb2ad6f0586fe00927ae2da1299738a82ccd86af713717c905444257R4-R7